withdrawn-packages.txt

```diff
@@ -1 +1,4 @@
-pgvector-0.6.1-r0.apk
\ No newline at end of file
+pgvector-0.6.1-r0.apk
+python3-3.11.1-r3
```

Comment:
```suggestion
python3-3.11.1-r3.apk
```

Reply:
this does look correct, given pgvector having the .apk on the end.
@xnox was there a reason you did not add .apk to the new entriews?